MeshLib/Mesh.cpp

```diff
@@ -269,5 +269,15 @@ void Mesh::setNodesConnectedByElements()
 	}
 }
 
+std::size_t Mesh:: getMaximumNConnectedNodesToNode() const
+{
+	std::size_t max_ncn = 0;
+	for(auto node : _nodes)
+	{
+		if(node->getConnectedNodes().size() > max_ncn)
+			max_ncn = node->getConnectedNodes().size();
+	}
+	return max_ncn + 1;
+}
```

Comment:
do you actually *need* this method here? I cannot find a call to this function anywhere in the code. but if you are going to need it only once or twice it makes more sense to implement it where you need it instead of enlarging the general mesh-interface. ( also, I think you could implement this more efficiently using ```std::max_element()``` )